roles/httpd/tasks/main.yml

```diff
@@ -1,30 +1,21 @@
 ---
-- name: Install Apache httpd
-  ansible.builtin.package:
-    name:
-      - httpd
-      - mod_ssl
+- name: Pull Apache container image
+  containers.podman.podman_image:
+    name: "{{ httpd_container_image }}"
     state: present
 
-- name: Set httpd_can_network_connect so Apache can connect to Puma and Gunicorn
-  ansible.posix.seboolean:
-    name: httpd_can_network_connect
-    state: true
-    persistent: true
-
-- name: Disable welcome page
-  ansible.builtin.file:
-    path: /etc/httpd/conf.d/welcome.conf
-    state: absent
-
-- name: Create cert directories
+- name: Create required directories
   ansible.builtin.file:
     path: "{{ item }}"
     state: directory
+    owner: root
+    group: root
     mode: "0750"
   loop:
     - "{{ httpd_ssl_dir }}/certs"
     - "{{ httpd_ssl_dir }}/private"
+    - "{{ httpd_conf_dir }}"
+    - "/var/log/httpd"
```

Comment:
Rather than copying this path in multiple places, can you introduce `httpd_log_dir`?